DOCUMENTATION.md

```diff
@@ -321,14 +334,14 @@ This object is returned by the method `get()` of the ApplicationClient class. He
 The class constructor can be called following this scheme:
 
 ```python
-HandlerClient(app_id, access_key_or_token, discovery_address="discovery.thethings.network:1900", certificate="")
+HandlerClient(app_id, access_key_or_token, discovery_address="discovery.thethings.network:1900", certificate_path="")
 ```
 
 * `app_id`: **string** this the name given to the application when it was created.
   ![Screenshot of the console with app section](./images/app-console.png?raw=true)
 * `app_access_key`: **string** this can be found at the bottom of the application page under **ACCESS KEYS**. The key needs the `settings`authorization.
 * `discovery_address`: **string** this is the address of the discovery server to use in order to find back the address of the handler to which the application in registered, in the `{hostname}:{port}` format.
-* `certificate`: **string** this is the path to the certificate used to connect in a secure way to the discovery server.
+* `cert_path`: **string** this is the path to the certificate used to connect in a secure way to the discovery server.
```

Comment:
`cert_path` or `certificate_path` (l.337)? Make the parameter name consistent with `ApplicationClient`